src/main/java/io/zeebe/monitor/rest/ExceptionHandler.java

```diff
@@ -0,0 +1,53 @@
+package io.zeebe.monitor.rest;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+@ControllerAdvice
+public class ExceptionHandler {
+
+	private static final Logger LOG = LoggerFactory.getLogger(ExceptionHandler.class);
+
+	private final String basePath;
+	private final String logoPath;
+	private final String customCssPath;
+	private final String customJsPath;
+	private final String customTitle;
+
+
+	public ExceptionHandler(@Value("${server.servlet.context-path}") final String basePath,
+	                        @Value("${white-label.logo.path}") final String logoPath,
+	                        @Value("${white-label.custom.title}") final String customTitle,
+	                        @Value("${white-label.custom.css.path}") final String customCssPath,
+	                        @Value("${white-label.custom.js.path}") final String customJsPath) {
+		this.basePath = basePath.endsWith("/") ? basePath : basePath + "/";
+		this.logoPath = logoPath;
+		this.customTitle = customTitle;
+		this.customCssPath = customCssPath;
+		this.customJsPath = customJsPath;
+
+	}
```

Comment:
In order to reduce the duplication, it would be nice to bundle this configuration into an object that can be used in both cases. 

Do you want to refactor this part?

Reply:
Will do.